site/en/workflows.md

```diff
@@ -21,6 +21,8 @@ At first, the image should be prepared for OCR.
 Optionally, you can start off your workflow by enhancing your images, which can be vital for the following binarization. In this processing step,
 the raw image is taken and enhanced by e.g. grayscale conversion, brightness normalization, noise filtering, etc.  
 
+**Note:** `ocrd-image-preprocess` converts or enhances images and can thereby replace some processors described in the following workflow steps
```

Comment:
`ocrd-image-preprocess` can be used to run arbitrary preprocessing commands on images or AlernativeImages and can be seen as a generic replacement for many of the following workflow steps, provided a matching external tool exists.